src/main/java/hudson/plugins/doclinks/artifacts/ArtifactsDocLinksConfig.java

```diff
@@ -175,7 +175,11 @@ public FormValidation doCheckTitle(@QueryParameter String value) {
                         break;
                     }
                     File dir = b.getArtifactsDir();
-                    if (!dir.exists() || !dir.isDirectory() || dir.listFiles().length <= 0) {
+                    if (!dir.exists() || !dir.isDirectory()) {
+                        continue;
+                    }
+                    File[] files = dir.listFiles();
+                    if (files == null || files.length <= 0) {
                         continue;
                     }

```

Comment:
<BugInstance type='NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE' priority='Normal' category='STYLE' message='Possible null pointer dereference in hudson.plugins.doclinks.artifacts.ArtifactsDocLinksConfig$DescriptorImpl.getBuildsToCheckArtifacts(AbstractProject) due to return value of called method' lineNumber='178'/>

Actually, this never be `null` as dir is a directory. (`File#listFiles` returns `null` only when it is not a directory)